src/test/java/org/openrewrite/kotlin/.editorconfig

```diff
@@ -2,4 +2,5 @@ root = true
 
 [*.java]
 indent_size = 4
-ij_continuation_indent_size = 2
+ij_continuation_indent_size = 4
+# ij_continuation_indent_size = 8
```

Comment:
I added a line when applying formatting to TypeMapping and SignatureBuilding tests.